apix-rest-v1/src/main/java/eu/xenit/apix/rest/exceptions/FileExistsException.java

```diff
@@ -0,0 +1,54 @@
+package eu.xenit.apix.rest.exceptions;
+
+import eu.xenit.apix.data.NodeRef;
+
+public class FileExistsException extends RuntimeException {
```

Comment:
(non-blocking) I would like to clean this class up a bit:
- make fields private
- remove unused methods (such as setters and unused ctors)

Reply:
im not convinced removing the ctors is a good idea, seeing as exceptions follow a pretty standard pattern. Not having the expected ones is at best an annoyance, at worst an active encouragement not to use the exception.
I put this one in with an eye on possibly removing alfresco dependencies from the interface and rest module. That would imply that the apix services will have to also use this (and other) exceptions, Which in turn would lead to people having to interact with these exceptions directly. Ergo the need for the standard ctors.

Reply:
I can see your point.
Was non-blocking for me anyway, so I'll leave decision to you.